common/metrics/defs.go

```diff
@@ -1265,21 +1265,23 @@ const (
 	SequentialTaskQueueProcessingLatency
 	SequentialTaskTaskProcessingLatency
 
+	HistoryArchiverArchiveNonRetryableErrorCount
+	HistoryArchiverArchiveTransientErrorCount
 	HistoryArchiverHistoryMutatedCount
-	HistoryArchiverTotalUploadSize
+	HistoryArchiverUploadedHistorySize
+	HistoryArchiverHistorySize
 
 	// The following metrics are only used by internal archiver implemention.
 	// TODO: move them to internal repo once cadence plugin model is in place.
 	HistoryArchiverBlobExistsCount
+	HistoryArchiverUploadedBlobSize
```

Comment:
rename: HistoryArchiverBlobSize - it might be the size of a blob which we did not upload